extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -428,8 +428,20 @@ export class SchemaCompareMainWindow {
 						// show the first dependent that caused this to fail in the warning message
 						const diffEntryName = this.createName(diff.sourceValue ? diff.sourceValue : diff.targetValue);
 						const firstDependentName = this.createName(result.blockingDependencies[0].sourceValue ? result.blockingDependencies[0].sourceValue : result.blockingDependencies[0].targetValue);
-						const cannotExcludeMessage = localize('schemaCompare.cannotExcludeMessage', "Cannot exclude {0}. Included dependents exist such as {1}", diffEntryName, firstDependentName);
-						const cannotIncludeMessage = localize('schemaCompare.cannotIncludeMessage', "Cannot include {0}. Excluded dependents exist such as {1}", diffEntryName, firstDependentName);
+						let excludeMsg;
+						if (firstDependentName) {
```

Comment:
You can just combine these two if's into a single statement that sets both excludeMsg and includeMsg

Reply:
That makes sense. I will do that.

As for updating the message with a comma, I also feel there should be a "." at the end, that kind of marks the end of the message. But most of the messages that I checked, didn't have one. I will check with PMs on this.

Reply:
I agree about the periods, but sticking with how it's been done elsewhere is fine there too since that doesn't really affect the grammatical reading of it.

Reply:
Sounds good. 
I will run through the PMs for the comma thing.